sdks/python/hatchet_sdk/utils/aio_utils.py

```diff
@@ -1,83 +1,4 @@
 import asyncio
-import inspect
-from concurrent.futures import Executor
-from functools import partial, wraps
-from typing import Any
-
-
-## TODO: Stricter typing here
-def sync_to_async(func: Any) -> Any:
```

Comment:
for discussion - I don't think we should encourage this. IMO, people should use `asyncio.to_thread` directly and we should have guidance on that (see asyncio docs), but I don't think we should expose an automagic method that will "just make sync things run async"

one other option is to whittle this down to basically just being a `to_thread` wrapper, but at that point, might as well just use `to_thread` I think